react/src/components/forms/InputCurrency/index.js

```diff
@@ -140,15 +140,15 @@ const Currency = (props) => {
           const handleBlur = (e) => {
             const { type } = e;
             const inputEl = ref.current;
-            const stringValue = inputEl.value;
-            // isNotNumber returns true if stringValue is null, undefined or 'NaN'
+            const { value } = inputEl;
```

Comment:
If `ref.current` is null, then this line will error out, as you can't destructure null.